docroot/themes/humsci/humsci_basic/src/scss/components/_block-layout.scss

```diff
@@ -54,13 +54,20 @@
 }
 
 .email {
+  word-break: break-all;
+
   a {
     @include hb-link--inline($icon: false);
   }
 
   [class^="fa-"] {
     display: none;
   }
+
+  //Removing break-all from the email label only, to avoid contact column titles from wrapping unintentionally.
+  & .field-label {
+    word-break: normal;
+  }
```

Comment:
This was added to prevent this type of unexpected label wrapping:
![image](https://user-images.githubusercontent.com/9355698/98156888-7a49e680-1ea6-11eb-84d5-f4948a9863f7.png)